docs/guides/writing_stubs.rst

```diff
@@ -712,3 +775,99 @@ all type checkers::
 
   def foo(x: int | str) -> int | None: ...  # recommended
   def foo(x: Union[int, str]) -> Optional[int]: ...  # ok
+
+.. _using-any:
+
+Using `Any` and `object`
```

Comment:
Also need a bunch more double backticks here.

(Single backticks in RST use the "default role" which we should avoid. We want linting against it.)